spec/lib/middlewares/reject_invalid_request_uri_spec.rb

```diff
@@ -6,10 +6,19 @@
     let(:app) { double('app') }
     let(:middleware) { RejectInvalidRequestUri.new(app) }
     let(:env) { { 'REQUEST_URI' => "/search?query=\xC0" } }
+    let(:response) { middleware.call(env) }
+    let(:status) { response.first }
 
-    it 'should return with status 400' do
-      status, header, body = middleware.call env
+    it 'returns a 400 status' do
       expect(status).to eq(400)
     end
+
+    context 'when invalid arguments are passed' do
```

Comment:
The bump to 2.5 resulted in `simplecov` identifying coverage missing for a condition in `RejectInvalidRequestUri`. This spec simply tests that conditional path.